provider-syllabus/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<name>Archiver Provider - Syllabus</name>
+	<description>An archiver provider for the syllabus tool</description>
+	<groupId>org.sakaiproject.archiver.provider</groupId>
+	<artifactId>archiver-provider-syllabus</artifactId>
+
+	<packaging>sakai-component</packaging>
+
+	<properties>
+    	<deploy.target>components</deploy.target>
+  	</properties>
+  	
+	<parent>
+		<groupId>org.sakaiproject.archiver</groupId>
+		<artifactId>archiver</artifactId>
+	    <version>0.2-SNAPSHOT</version>
+	</parent>
+
+  	<dependencies>
+		<dependency>
+			<groupId>org.sakaiproject.kernel</groupId>
+			<artifactId>sakai-kernel-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.kernel</groupId>
+			<artifactId>sakai-kernel-util</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.kernel</groupId>
+			<artifactId>sakai-component-manager</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.sakaiproject.archiver</groupId>
+			<artifactId>archiver-api</artifactId>
+			<version>0.2-SNAPSHOT</version>
```

Comment:
FYI this version can now be removed as it is in the parent pom and will be inherited.